app/controllers/search_controller.rb

```diff
@@ -1,13 +1,24 @@
 class SearchController < SessionsController
   before_action :current_user
-  # before_action :signed_in
+  before_action :category, only: [:featured]
   require 'will_paginate/array'
 
   def explore
     @repositories = Repository.paginate(:per_page=>12,:page=>params[:page]).order([sort_order].to_h).page params[:page]
     @photos = photo_hash
   end
 
+  def equipment
```

Comment:
in show repository you can click on any of the equipment tags and it renders a search view with no results so I added this to make it work